src/Mod/AddonManager/addonmanager_macro.py

```diff
@@ -1,38 +1,73 @@
 # -*- coding: utf-8 -*-
-#***************************************************************************
-#*                                                                         *
-#*   Copyright (c) 2018 Gaël Écorchard <galou_breizh@yahoo.fr>             *
-#*                                                                         *
-#*   This program is free software; you can redistribute it and/or modify  *
-#*   it under the terms of the GNU Lesser General Public License (LGPL)    *
-#*   as published by the Free Software Foundation; either version 2 of     *
-#*   the License, or (at your option) any later version.                   *
-#*   for detail see the LICENCE text file.                                 *
-#*                                                                         *
-#*   This program is distributed in the hope that it will be useful,       *
-#*   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
-#*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
-#*   GNU Library General Public License for more details.                  *
-#*                                                                         *
-#*   You should have received a copy of the GNU Library General Public     *
-#*   License along with this program; if not, write to the Free Software   *
-#*   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
-#*   USA                                                                   *
-#*                                                                         *
-#***************************************************************************
-
-## @package AddonManager_macro
-#  \ingroup ADDONMANAGER
-#  \brief Unified handler for FreeCAD macros that can be obtained from different sources
+
+"""
+***************************************************************************
+*                                                                         *
+*   Copyright (c) 2018 Gaël Écorchard <galou_breizh@yahoo.fr>             *
```

Comment:
This copyright notice should not be inside triple quotes.

Reply:
reverted